### ISSUE TITLE
Realtime database looks empty

### ISSUE THREAD

**Issue by jof4002** [jsayol/vscode-firebase-explorer#30]
I setup realtime database for my project.
But firebase explorer says Database is empty for this project.

Is there anything I can do to fix this problem ?

**Comment by DT1000101** [comment]
I am having the same problem, but not in all of my projects.
The only correlation that I have noticed so far is that all the projects with the Database location set to `(us-central1)` seem to work while the ones in `(europe-west1)` do not. Not sure if this is a coincidence though.
Also, the extension only displays anything when the root path has one or more children. I.e. simply having `your-proj-name-rtdb : null` will not show anything in the extension, you need to add something to it.

**Comment by jof4002** [comment]
Yes. our DB location set to (asia-southeast1). This seems to be the cause of the problem,
Thanks for that info!
